src/simulator/cube.py

```diff
@@ -11,29 +10,30 @@ def __init__(self, length: float, height: float, width: float):
         self.height = height
         self.width = width
 
-    def bounding_box(self) -> Box:
+    def bounding_box(self, position: Position) -> Box:
```

Comment:
@mehdi138r Can you add a comment, describing what is happening in this function? e.g. What's the role of these devisions by 2?